src/main/java/com/strandls/cca/service/impl/CCADataServiceImpl.java

```diff
@@ -688,4 +708,92 @@ public Boolean sendPermissionGrant(HttpServletRequest request, EncryptedKey encr
 		return false;
 	}
 
+
+	@Override
+	public Map<String, Object> searchCCAData(String query, HttpServletRequest request, UriInfo uriInfo)
```

Comment:
the dao class also has `searchCCAData`  as the function name. Can we rename that to something like `getSearchCCAData` (or something to differentiate that function name from this just to avoid confusion)

Reply:
done